api_tests/scripts/apigee_api.py

```diff
@@ -29,36 +29,74 @@ def _generate_uuid():
         unique_id = uuid.uuid4()
         return str(unique_id)
 
+    def check_status_code(self, response: 'response type', expected_status_code: int) -> bool:
+        """Compare the actual and expected status code for a given response"""
+        self._validate_response(response)
+        self._verify_status_code(expected_status_code)
+        return response.status_code == expected_status_code
+
+    @staticmethod
+    def _validate_response(response: 'response type') -> None:
+        """Verifies the response provided is of a valid response type"""
+        if not type(response) == requests.models.Response:
+            raise TypeError("Expected response type object for response argument")
+
+    @staticmethod
+    def _verify_status_code(status_code: int or str) -> None:
```

Comment:
We don't need any of these validations in here. Request object has a fix API. This method can be deleted and we can just use a inline boolean comparison.